### PR TITLE
Add minisim extension

### PR DESCRIPTION
**Pull request by okwasniewski**
## Description

New extension: Allows to control [Minisim](https://github.com/okwasniewski/MiniSim) small app to launch and manage android and ios simulators.

## Screencast

https://github.com/raycast/extensions/assets/52801365/8bb8970c-79e3-418b-88fd-9da706c7d009

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/minisim/src/actions.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { runAppleScript } from "run-applescript";
```
Comment (pernielsentikaer) [inline extensions/minisim/src/actions.ts:1]: You should load `runAppleScript` from `@raycast/utils` instead

**Comment by pernielsentikaer** [comment]
Could you remove your yarn.lock and add a package-lock instead 🙂

File: extensions/minisim/src/actions.ts (hunk 1 above)
Comment (okwasniewski) [reply extensions/minisim/src/actions.ts]: Done 👍🏻

**Comment by okwasniewski** [comment]
Hey @pernielsentikaer, thanks for reviewing! 

I've applied your comments

**Review by pernielsentikaer — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Could you add [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) as the last thing 🙂

**Comment by okwasniewski** [comment]
@pernielsentikaer Done 👍🏻

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by okwasniewski** [comment]
Great, what are the next steps? Only people with write access can merge PRs 😅

**Pull request merged**